packages/asset-service/src/service/AssetService.ts

```diff
@@ -1,6 +1,12 @@
+import { caip19 } from '@shapeshiftoss/caip'
 import { Asset, AssetDataSource, BaseAsset, ChainTypes, NetworkTypes } from '@shapeshiftoss/types'
 import axios from 'axios'
 
+/* eslint-disable */
+const Identicon = require('identicon.js')
+const { fonts, renderPixels } = require('js-pixel-fonts')
```

Comment:
If possible, it would be better to use ESM. I've checked and none of these libraries support it so we're effectively force to use require statements.

Reply:
Yes, both of these libraries are not supporting ESM so we currently can only require them.